modules/utils/getNpmPackageInfo.js

```diff
@@ -7,6 +7,11 @@ function cleanPackageConfig(packageConfig) {
   return {
     name: packageConfig.name,
     version: packageConfig.version,
+    unpkg: packageConfig.unpkg,
+    main: packageConfig.main,
+    module: packageConfig.module,
+    "js:next": packageConfig["js:next"],
```

Comment:
this is awesome, great catch on adding the `js:next` as well. I'd anticipate having to replace the double quotes with single quotes, though; it looks like there was recently [a big refactor](https://github.com/unpkg/unpkg.com/commit/19b2e5574b5ab5cb711ddbf27c5300d114bed4dc)